shared-test/src/main/java/uk/gov/di/authentication/sharedtest/basetest/HandlerIntegrationTest.java

```diff
@@ -187,13 +187,7 @@ public String getTxmaAuditQueueUrl() {
                     ipvPrivateKeyJwtSigner,
                     spotQueue,
                     docAppPrivateKeyJwtSigner,
-                    configurationParameters) {
-
-                @Override
-                public boolean isAuthOrchSplitEnabled() {
-                    return true;
-                }
-            };
+                    configurationParameters) {};
```

Comment:
Can this all be removed if we are just using `TEST_CONFIGURATION_SERVICE` for the the integration test ?